sdk/synapse/synapse-access-control/package.json

```diff
@@ -9,7 +9,8 @@
   "dependencies": {
     "@azure/core-paging": "^1.1.1",
     "@azure/core-http": "^1.2.0",
-    "tslib": "^2.0.0"
+    "tslib": "^2.0.0",
+    "rimraf": "^3.0.0"
```

Comment:
@willmtemple hmmm, I think you meant to add it to `devDependencies` instead, no?

Reply:
I fixed that here: https://github.com/Azure/azure-sdk-for-js/pull/12940